packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -279,6 +279,58 @@ export class CloudAPIClient {
         const data = await res.json()
         return data['token']
     }
+
+    /** Polls MRT for deployment status every 30 seconds. */
+    async waitForDeploy(project: string, environment: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            /** Milliseconds to wait between checks. */
+            const delay = 30e3
```

Comment:
Readability. I had to look up `30e3` I prefer to avoid code that makes you have to look things up unless it's necessary

Reply:
+1 on this. I had to look this up as well when I first saw it